app/models/customer.rb

```diff
@@ -0,0 +1,3 @@
+class Customer < ApplicationRecord
```

Comment:
Yea, since we are using should matchers in testing, I usually use it in the model as well